### DISCUSSION TITLE
Only front camera issue

### DISCUSSION THREAD

**Discussion by sweerties** [community/community#56257] [GitHub Education]
### Hi everyone,

Hello to everyone. Today I received a student certificate for the Github Student Pack from my school. When I get to the current page to upload it, I can't shoot the document because my front camera shows in both options. When I shoot, the quality is very low. Please help me.

**Comment by thomashirtz** [comment]
I have the same issue.

**Reply by jackyyy65** [reply]
I'm sorry to hear that you're having difficulty uploading your student certificate due to the low quality of your front camera. As an AI text-based model, I'm unable to directly assist with camera issues, but I can [offer](https://asphaltnitro.download/) some general suggestions. First, check your camera settings to see if there are any options to improve image quality. Additionally, try adjusting the lighting conditions or finding a well-lit area when capturing the document. If the problem persists, you may want to consider using an external camera or borrowing a friend's device with a better camera for the upload.

**Comment by samus-aran** [comment]
Hi @sweerties 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Comment by Messjer** [comment]
I have the same issue and submitted and ticket too.

**Comment by gabrielqueeiroz** [comment]
I have the same issue, hope they fix it soon

**Comment by ashutoshrai387** [comment]
It been so much long but nothing,no action was taken on this issue. Sad.
